### PR TITLE
nfs volume fixes for new kubernetes API

### PR DESCRIPTION
**Pull request by blankdots**
### Describe the pull request:

- [x] Bug fix

### Pull request long description:

The NFSVolume API in kubernetes changed and this required some fixes to make it work.

### Changes made:

1. updated CEGA MQ to work with custom PVC
2. updated e2e tester, Apache Mina inbox, RES, Verify and Ingest with new NFS volume k8s syntax
3. need to update the key in DOA because `LocalEGA-deploy-init` changed I the helm lint failed.

### CODE REVIEW

File: ega-charts/localega/templates/doa-secret.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-{{- $c4gh_key := .Files.Glob "config/key.1.c4gh.sec" }}
+{{- $c4gh_key := .Files.Glob "config/ega_key.c4gh.sec" }}
 apiVersion: v1
 kind: Secret
 metadata:
   name: {{ template "localega.name" . }}-doa-secret
 data:
 {{ ( $c4gh_key ).AsSecrets | trim | indent 2 }}
-  key.pass: {{ .Values.secrets.c4gh_passphrase | quote | trimall "\"" | b64enc }}
+  key.pass: {{ .Values.secrets.ega_c4gh_passphrase | quote | trimall "\"" | b64enc }}
```
Comment (jbygdell) [inline ega-charts/localega/templates/doa-secret.yaml]: This change is not related to NFS

File: ega-charts/localega/templates/podsecuritypolicy.yaml [hunk 2]
```diff
@@ -40,4 +40,7 @@ spec:
   - configMap
   - emptyDir
   - projected
+  {{ if and (eq "nfs" .Values.ingest.persistence.storageClass) (eq "FileStorage" .Values.config.data_storage_type) }}
```
Comment (jbygdell) [inline ega-charts/localega/templates/podsecuritypolicy.yaml]: No need for the if statement

**Review by jbygdell — DISMISSED** [review]

File: ega-charts/localega/templates/doa-secret.yaml (hunk 1 above)
Comment (blankdots) [reply ega-charts/localega/templates/doa-secret.yaml]: true, but the helm linter will fail because the deploy-init latest is used, and it cannot find that key, unless I wait for another branch to be merged first and rebase on that.

File: ega-charts/localega/templates/podsecuritypolicy.yaml (hunk 2 above)
Comment (blankdots) [reply ega-charts/localega/templates/podsecuritypolicy.yaml]: removed.

File: ega-charts/localega/templates/doa-secret.yaml (hunk 1 above)
Comment (blankdots) [reply ega-charts/localega/templates/doa-secret.yaml]: rebased to milestone_8

**Review by viklund — APPROVED** [review]

**Review by jbygdell — APPROVED** [review]
